pkgs/tools/security/cryptomator/default.nix

```
@@ -0,0 +1,89 @@
+{ stdenv, lib, fetchurl, fetchFromGitHub
+, autoPatchelfHook
+, appimageTools
+, desktop-file-utils
+, fuse, packer
+, maven, openjdk14, openjfx14, makeWrapper, glib, wrapGAppsHook
+}:
+
+let
+  pname = "cryptomator";
+  version = "1.5.11";
+
+  src = fetchFromGitHub {
+    owner = "cryptomator";
+    repo = "cryptomator";
+    rev = "${version}";
+    sha256 = "0sr6466ff8g74ygbmnmllz52x8cgx1lb14ggdfj6vy9pxiga7zfh";
+  };
+
+  # perform fake build to make a fixed-output derivation out of the files downloaded from maven central (120MB)
+  deps = stdenv.mkDerivation {
+    name = "cryptomator-${version}-deps";
+    inherit src;
```

for readability i would add a newline:

```suggestion
    inherit src;

```